Session.js

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import SessionService from './SessionService';
```

Comment:
What is the value of this Session class?  It doesn't look like it's saving state or anything.  Why not access the service directly?